2024-11-27 07:49:45,756 INFO    Thread-11 :593242 [wandb_setup.py:_flush():77] Current SDK version is 0.18.1
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_setup.py:_flush():77] Configure stats pid to 593242
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_setup.py:_flush():77] Loading settings from /home/sjl356/.config/wandb/settings
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_setup.py:_flush():77] Loading settings from /home/sjl356/badrobotsIRL/training/binary_models/wandb/settings
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_setup.py:_flush():77] Loading settings from environment variables: {'api_key': '***REDACTED***', 'project': 'lstm_all_v0', 'entity': 'shannon-jliu', 'root_dir': '/home/sjl356/badrobotsIRL/training/binary_models', 'run_id': 'ca154zgu', 'sweep_param_path': '/home/sjl356/badrobotsIRL/training/binary_models/wandb/sweep-zt21q239/config-ca154zgu.yaml', 'sweep_id': 'zt21q239'}
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_setup.py:_flush():77] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program_relpath': 'training/binary_models/lstm_model_wandb.py', 'program_abspath': '/home/sjl356/badrobotsIRL/training/binary_models/lstm_model_wandb.py', 'program': '/home/sjl356/badrobotsIRL/training/binary_models/lstm_model_wandb.py'}
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_init.py:_log_setup():532] Logging user logs to /home/sjl356/badrobotsIRL/training/binary_models/wandb/run-20241127_074945-ca154zgu/logs/debug.log
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_init.py:_log_setup():533] Logging internal logs to /home/sjl356/badrobotsIRL/training/binary_models/wandb/run-20241127_074945-ca154zgu/logs/debug-internal.log
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_init.py:init():616] calling init triggers
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_init.py:init():623] wandb.init called with sweep_config: {'activation_function': 'relu', 'batch_size': 32, 'dense_units': 64, 'dropout_rate': 0.8, 'epochs': 500, 'fusion_type': 'early', 'learning_rate': 0.01, 'loss': 'categorical_crossentropy', 'lstm_units': 128, 'modality': 'combined', 'num_lstm_layers': 2, 'optimizer': 'rmsprop', 'recurrent_regularizer': 'l1', 'sequence_length': 30, 'use_bidirectional': True, 'use_norm': False, 'use_pca': True, 'use_stats': False}
config: {}
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_init.py:init():666] starting backend
2024-11-27 07:49:45,757 INFO    Thread-11 :593242 [wandb_init.py:init():670] setting up manager
2024-11-27 07:49:45,762 INFO    Thread-11 :593242 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-27 07:49:45,763 INFO    Thread-11 :593242 [wandb_init.py:init():678] backend started and connected
2024-11-27 07:49:45,772 INFO    Thread-11 :593242 [wandb_run.py:_config_callback():1388] config_cb None None {'activation_function': 'relu', 'batch_size': 32, 'dense_units': 64, 'dropout_rate': 0.8, 'epochs': 500, 'fusion_type': 'early', 'learning_rate': 0.01, 'loss': 'categorical_crossentropy', 'lstm_units': 128, 'modality': 'combined', 'num_lstm_layers': 2, 'optimizer': 'rmsprop', 'recurrent_regularizer': 'l1', 'sequence_length': 30, 'use_bidirectional': True, 'use_norm': False, 'use_pca': True, 'use_stats': False}
2024-11-27 07:49:45,778 INFO    Thread-11 :593242 [wandb_init.py:init():773] updated telemetry
2024-11-27 07:49:45,966 INFO    Thread-11 :593242 [wandb_init.py:init():806] communicating run to backend with 90.0 second timeout
2024-11-27 07:49:46,137 INFO    Thread-11 :593242 [wandb_init.py:init():857] starting run threads in backend
2024-11-27 07:49:46,436 INFO    Thread-11 :593242 [wandb_run.py:_console_start():2459] atexit reg
2024-11-27 07:49:46,436 INFO    Thread-11 :593242 [wandb_run.py:_redirect():2307] redirect: wrap_raw
2024-11-27 07:49:46,436 INFO    Thread-11 :593242 [wandb_run.py:_redirect():2372] Wrapping output streams.
2024-11-27 07:49:46,437 INFO    Thread-11 :593242 [wandb_run.py:_redirect():2397] Redirects installed.
2024-11-27 07:49:46,437 INFO    Thread-11 :593242 [wandb_init.py:init():900] run started, returning control to user process
2024-11-27 11:02:15,162 ERROR   Thread-1  :593242 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 11:02:15,163 ERROR   Thread-1  :593242 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 11:12:58,478 ERROR   Thread-1  :593242 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 11:12:58,478 ERROR   Thread-1  :593242 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 11:19:33,394 ERROR   Thread-1  :593242 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 11:19:33,394 ERROR   Thread-1  :593242 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 13:24:37,024 ERROR   Thread-1  :593242 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 13:24:37,024 ERROR   Thread-1  :593242 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 13:27:38,839 ERROR   Thread-1  :593242 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 13:27:38,840 ERROR   Thread-1  :593242 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 13:55:19,589 INFO    Thread-11 :593242 [wandb_run.py:_finish():2158] finishing run shannon-jliu/lstm_all_v0/ca154zgu
2024-11-27 13:55:19,589 INFO    Thread-11 :593242 [wandb_run.py:_atexit_cleanup():2422] got exitcode: 0
2024-11-27 13:55:19,590 INFO    Thread-11 :593242 [wandb_run.py:_restore():2404] restore
2024-11-27 13:55:19,590 INFO    Thread-11 :593242 [wandb_run.py:_restore():2410] restore done
2024-11-27 13:55:21,297 INFO    Thread-11 :593242 [wandb_run.py:_footer_history_summary_info():4037] rendering history
2024-11-27 13:55:21,298 INFO    Thread-11 :593242 [wandb_run.py:_footer_history_summary_info():4069] rendering summary
2024-11-27 13:55:21,307 INFO    Thread-11 :593242 [wandb_run.py:_footer_sync_info():3996] logging synced files
